src/AerosolActivation.jl

```diff
@@ -81,6 +90,19 @@ function mean_hygroscopicity(param_set::APS, ad::AM.AerosolDistribution)
         nom / den * _molmass_water / _ρ_cloud_liq
     end
 end
+function mean_hygroscopicity_parameter(
+    param_set::APS,
+    ad::AM.AerosolDistribution{NTuple{N, T}},
+) where {N, T <: AM.Mode_κ}
+
+    return ntuple(length(ad.Modes)) do i
+
+        modei = ad.Modes[i]
+        sum(1:(modei.n_components)) do i
+            modei.vol_mix_ratio[i] * modei.kappa[i]
+        end
+    end
```

Comment:
I would avoid using a nested `do` loop with both indices being the same--at least for clarity since the behavior depends on the scope rules.

Reply:
good catch! Thanks!